### DISCUSSION TITLE
Front End (User interface) for sign-up feature

### DISCUSSION THREAD

**Discussion by estefania-2817** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#61] [Ideas]
Create a proposal for the sign-up user interface

Description:

- Build the user interface of the sign up page for students and organizations.
- In case of student, build the basic structure of the sign up form where the user provides their student id, email, password and password confirmation.
- In case of organization, build the basic structure of the sign up form where the organization provides the name of organization, email, password and password confirmation.
- Create the buttons where the each of them can submit their solicited information.
- Include the link in the case if the user or organization already has been registered and only needs to log in.

Requirements:
For students, create the fields of:
- student id
- email
- password
- password confirmation
- log in link
- submit(Create account) button.
For organizations, create the fields of:
- organization name
- email
- password
- password confirmation
- log in link
- submit(Create account) button.

**Comment by estefania-2817** [comment]
![image](https://github.com/user-attachments/assets/4932085f-a285-4b7f-8956-06511b25f259)

**Reply by AngelUPR** [reply]
I think it would be best if we made the create a student account and organization as two buttons, and after we click one of those, the corresponding prompt would appear instead of having both appear at the same time.

I would also leave the "log in instead" bottom in the prompt, but remove the Login at the top of the page
